Packs/CommonPlaybooks/Playbooks/playbook-CVE_Enrichment_-_Generic_v2.yml

```diff
@@ -97,178 +104,55 @@ tasks:
-    type: condition
-    task:
-      id: 8f3572d2-dbfc-4c4b-82af-94d06b99e727
-      version: -1
-      name: Is "CVE Search" integration enabled?
-      description: Verify that there's a valid instance of "CVE Search" enabled.
-      type: condition
-      iscommand: false
-      brand: ''
-    nexttasks:
-      '#default#':
-      - '2'
-      'yes':
-      - '6'
-    separatecontext: false
-    conditions:
-    - label: 'yes'
-      condition:
-      - - operator: isExists
-          left:
-            value:
-              complex:
-                root: modules
-                filters:
-                - - operator: containsGeneral
-                    left:
-                      value:
-                        simple: modules.brand
-                      iscontext: true
-                    right:
-                      value:
-                        simple: CVE Search
-                    ignorecase: true
-                - - operator: isEqualString
-                    left:
-                      value:
-                        simple: modules.state
-                      iscontext: true
-                    right:
-                      value:
-                        simple: active
-                    ignorecase: true
-                accessor: brand
-            iscontext: true
-    view: |-
-      {
-        "position": {
-          "x": 970,
-          "y": 790
-        }
-      }
-    note: false
-    timertriggers: []
-    ignoreworker: false
-    skipunavailable: false
-    quietmode: 0
   "6":
     id: "6"
-    taskid: c08adb3a-b8c3-4a9b-8290-2f6dcda45d88
+    taskid: 18920167-4098-40d7-8781-16125c58e9c9
     type: regular
     task:
-      id: c08adb3a-b8c3-4a9b-8290-2f6dcda45d88
+      id: 18920167-4098-40d7-8781-16125c58e9c9
       version: -1
-      name: Get CVE info from "CVE Search"
+      name: Get vulnerability details by CVE ID
       description: Returns CVE information by CVE ID.
       script: '|||cve'
       type: regular
       iscommand: true
-      brand: CVE Search
+      brand: ""
     nexttasks:
       '#none#':
       - '2'
     scriptarguments:
-      cve_id:
+      cve:
         complex:
           root: inputs.cve_id
-    continueonerror: true
     separatecontext: false
     view: |-
       {
         "position": {
-          "x": 1160,
-          "y": 960
+          "x": 790,
+          "y": 650
         }
       }
     note: false
     timertriggers: []
     ignoreworker: false
+    continueonerrortype: ""
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
   '7':
     id: '7'
-    taskid: d2ff276a-dd81-4805-80c3-ee44976b4020
+    taskid: 3831c20e-46bd-4d20-8cc8-e64dc5addb6f
     type: regular
     task:
-      id: d2ff276a-dd81-4805-80c3-ee44976b4020
+      id: 3831c20e-46bd-4d20-8cc8-e64dc5addb6f
       version: -1
-      name: Get CVE information from XFE
+      name: Get vulnerability details by CVE ID
```

Comment:
Same name for 2 tasks, can we somehow distinguish between them?

Reply:
changed both tasks names to 'Get vulnerability details - cve-search command' and 'Get vulnerability details - cve command'. what do you think?

Reply:
Sounds good to me. Don't forget to change the pic as well :)